docs/front-end/frameworks-used.md

```diff
@@ -37,6 +37,62 @@ The following variables are required by the front end and should be setup as Git
 - `SENTRY_DSN`, the Sentry project Data Source Name (DSN)
 - `SENTRY_ORG`, the organisation specified in Sentry
 - `SENTRY_AUTH_TOKEN`, authentication token used for all communication with Sentry
+  
+### Adding specific local authority/council details
+
+The following variables are **required** by the front end to display specific local authority details such as customer service contact numbers and should be **set up as GitHub secrets for the repository**:
+
+- `CUSTOMER_SERVICES_TELEPHONE_NUMBER` -> The customer service number to contact ***during opening hours***
+- `OUT_OF_HOURS_CUSTOMER_SERVICES_TELEPHONE_NUMBER` -> The customer service number to contact ***outside of opening hours***
```

Comment:
`.` at the end?

Reply:
Done in [dac5a9d](https://github.com/Newarkandsherwood/housing-repairs-online/pull/19/commits/dac5a9ddf9287e66a74d39dee3808621e3349fb4)